syntax_and_semantics/literals/integers.md

```diff
@@ -1,8 +1,17 @@
 # Integers
 
-There are four signed integer types: [Int8](http://crystal-lang.org/api/Int8.html), [Int16](http://crystal-lang.org/api/Int16.html), [Int32](http://crystal-lang.org/api/Int32.html) and [Int64](http://crystal-lang.org/api/Int64.html), being able to represent numbers of 8, 16, 32 and 64 bits respectively.
-
-There are four unsigned integer types: [UInt8](http://crystal-lang.org/api/UInt8.html), [UInt16](http://crystal-lang.org/api/UInt16.html), [UInt32](http://crystal-lang.org/api/UInt32.html) and [UInt64](http://crystal-lang.org/api/UInt64.html).
+There are four signed integer types, and four unsigned integer types:
+
+Class | Length  | Minimum Value | Maximum Value
```

Comment:
Change column `Class` to `Type`. Ints are actually Crystal structs, but at this point in the docs it'd be confusing to point that out.